src/main/java/bio/terra/common/IamResourceTypeCODEC.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.common;
+
+import bio.terra.model.IamResourceTypeEnum;
+import bio.terra.service.auth.iam.IamResourceType;
+
+/**
```

Comment:
I though this might have been done accidentally, but I now see that Swagger CodeGen will merrily produce ENUMs with this problem.  We now have a coder/decoder for this.